dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import Dropzone from 'react-dropzone'
+import { AnchorButton, Intent } from '@blueprintjs/core'
+
+import { FormInput, TextInput, TextAreaInput } from '../inputs'
+
+require('../../../styles/components/person_form.scss')
+
+export default class PersonForm extends React.Component {
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      displayImg: null
+    }
+  }
+
+  onDrop(files) {
+    this.props.update('image', files[0])
+    this.setState({ displayImg: files[0].preview })
+  }
+
+  render() {
+    return (
+      <form onSubmit={e => e.preventDefault()}>
+        <FormInput
+          label='Full Name'
+          padded={false}
+          error={this.props.errors ? this.props.errors.full_name : null}>
+          <TextInput
+            placeholder='Full Name'
+            value={this.props.listItem.full_name || ''}
+            fill={true}
+            onChange={ e => this.props.update('full_name', e.target.value) } />
+        </FormInput>
+        <FormInput
+          label='Slug'
+          padded={false}
+          error={this.props.errors ? this.props.errors.slug : null}>
```

Comment:
for persons, the model allows blank for all the fields, so the server doesn't return errors.
should that be changed? or are there other error conditions?